contrib/ansible/roles/common/tasks/system-proxy.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Proxy | Install system http proxy environment variable
+  lineinfile: dest=/etc/environment regexp=^HTTP_PROXY= line=HTTP_PROXY="{{ system_http_proxy }}"
```

Comment:
I do know that kubectl isn't effected by this. I have been adding this to my setup for some time and decided to push it up.